ckanext/blob_storage/plugin.py

```diff
@@ -119,10 +121,12 @@ def get_blueprint(self):
     # IActions
 
     def get_actions(self):
+        authorizer = init_authorizer()
         return {
             'get_resource_download_spec': actions.get_resource_download_spec,
             'resource_schema_show': actions.resource_schema_show,
-            'resource_sample_show': actions.resource_sample_show
+            'resource_sample_show': actions.resource_sample_show,
+            'activity_resource_show': helpers.activity_resource_show
```

Comment:
I can't find any usages of this action. Are you using it anywhere or is this stale code?

Reply:
I removed this action.